frontend/app/work_packages/view_models/parent-relations-handler.js

```diff
@@ -28,7 +28,10 @@
 
 module.exports = function(CommonRelationsHandler, WorkPackageService, ApiHelper) {
   function ParentRelationsHandler(workPackage, parents, relationsId) {
-      var handler = new CommonRelationsHandler(workPackage, parents, relationsId);
+      var relations = parents.filter(function(parent) {
```

Comment:
@NobodysNightmare, I've tried to understand this to, spent about a day, and couldn't find the answer for it.